dataflow/src/org/checkerframework/dataflow/cfg/ControlFlowGraph.java

```diff
@@ -61,14 +61,18 @@
      */
     protected final List<ReturnNode> returnNodes;
 
+    /** Map from AST {@link Tree}s to generated {@link Tree}s. */
+    protected IdentityHashMap<Tree, List<Tree>> generatedTreesLookupMap;
```

Comment:
Can this field be `final`? I think the other non-final fields in this class can also be changed.